xbmc/video/dialogs/GUIDialogVideoInfo.cpp

```diff
@@ -2092,6 +2095,19 @@ bool CGUIDialogVideoInfo::OnGetFanart(const CFileItemPtr &videoItem)
     {
       int iFanart = 0;
       for (int i=0; i < movies.Size(); i++)
+      {
+        movies[i]->GetVideoInfoTag()->m_strPictureURL.Parse();
+        movies[i]->GetVideoInfoTag()->m_strPictureURL.GetThumbURLs(thumbs, "set.fanart", -1, true);
+      }
+      for (auto &fanart : thumbs)
+      {
+        CFileItemPtr item(new CFileItem(StringUtils::Format("fanart://Remote{0}", iFanart++), false));
```

Comment:
Is the capital hostname intended?

Reply:
Yup. These are short lived, gobbled up at [L2149](https://github.com/xbmc/xbmc/pull/13563/files/7087c07c0788f90bd796e1046832bcccf4499a0f#diff-dad57d70450e138307fd7397254d10d0R2149) and translated back to the actual URL of the image if the user selected one.

Reply:
Can you factor out the string duplication?

Reply:
Can do. Similar strings (starting with "fanart://" and "thumb://") are used for the same purpose in a few functions here, and then in a few other GUI windows, how far should I go? I could tackle all of it in a new PR, is there a good place to define them to be used across several source files?

Reply:
I suggest that "fanart://" and "thumb://"  strings are replaced (if we choose to do that) everywhere in one go in a separate PR.